src/index.js

```diff
@@ -31,6 +31,10 @@ export default class Undo {
     this.maxLength = maxLength || defaultOptions.maxLength;
     this.onUpdate = onUpdate || defaultOptions.onUpdate;
 
+    const buttonKey = /(Mac)/i.test(navigator.platform) ? 'metaKey' : 'ctrlKey';
+    this.shortcutUndo = `${buttonKey}+z`;
+    this.shortcutRedo = `${buttonKey}+y`;
```

Comment:
@MarioRodriguezS we can't fix the key `Z` and the key `Y` as constants. There are people that use different keyboard layouts and languages, our solution needs to extend to them as well